tools/print_psd_content.cpp

```diff
@@ -31,6 +31,10 @@ class Delegate : public psd::DecoderDelegate {
       psd::color_mode_string(fileHeader.colorMode));
   }
 
+  void onLayerSelected(const std::string &layerName){
+    std::printf("selected layer: %s\n", layerName.c_str());
```

Comment:
Prefer a first capital letter for comments and output messages if they were written with a capital letter.